packages/vats/src/lib-board.js

```diff
@@ -267,8 +267,8 @@ export const prepareBoardKit = baggage => {
          * `value` for its entire lifetime. For a well-known board, this is
          * essentially forever.
          *
-         * @param {PassableCap} value
-         * @throws if `value` is not a Key in the @agoric/store sense
+         * @param {RemotableObject} value
+         * @throws if `value` is not a ScalarKey in the @agoric/store sense
```

Comment:
I'm not sure `@agoric/store` has a `ScalarKey` sense. maybe "in the @endo/patterns sense"

Reply:
agreed. though this is one place ScalarKey ought to be mentioned, because this code only throws at runtime if it's it's not a ScalarKey. (it doesn't check that it's a remotable)

Reply:
I made it check with `isRemotable()`